clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/AbstractPartitionStateModel.java

```diff
@@ -168,7 +168,7 @@ public void onBecomeDroppedFromError(Message message, NotificationContext contex
   @Override
   public void reset() {
     try {
-      stopConsumption();
+      stopConsumption(false);
```

Comment:
What is the purpose of `reset`, again? I forgot when does Helix call this, so I've difficulty assessing whether updating or not updating the CV is ideal in this case...

Reply:
I see.

The PR will explicitly remove all the CV states during graceful shutdown.
I was hesitate to delete CV state in `reset` function since `reset` function can be invoked in different scenarios.

Reply:
I agree. It could have other scenarios calling reset in the HelixStateMachineEngine code and it would be safer to remove CVs only when we are sure to shutdown the server.